handbook/engineering/environments.md

```diff
@@ -0,0 +1,39 @@
+# Environment
+
+This page describes our different cloud environments.
+
+## Google Cloud
+
+We utilize multiple Google Cloud projects and folders to organize our workloads and manage access control for our engineers, as well as limit the scope of roles and service accounts across projects.
+
+### Root Projects
+These projects contain per-project permissions.
+
+- **TelligentSourcegraph**: Data pipeline and storage for pings and Cloud event logging
+- **Universities**: Sourcegraph instances for universities
+- **sourcegraph-interviews**: Shared project for interviews
+
+### Folders
+
+Dynamic or temporary projects are created in the folders listed below.
```

Comment:
Can you document what it means for a project to be "dynamic" or "temporary"?

Reply:
Ill remove this, it does not make sense anymore